integrationTests/chainSimulator/bridge/common.go

```diff
@@ -109,8 +114,36 @@ func deployBridgeSetup(
 	}
 }
 
-// Deposit will deposit tokens in the bridge sc safe contract
-func Deposit(
+func enshrineEsdtSafeContract(
```

Comment:
This is literally the same function as `simpleEnshrineEsdtSafeContract` , only called with a different argument for `DeployContract`

Reply:
You could've simply just passed from `deployBridgeSetup` contract address.

Reply:
Yes... fixed